ihorvath64-sandiego.md

```diff
@@ -0,0 +1,15 @@
+### San Diego
+
+#### Places to Eat
+- Jasmine Seafood Restaurant
```

Comment:
Jasmine is in Kearny Mesa - Excellent Dim Sum!
The Taco Stand is in La Jolla, Downtown and now also in Encinitas 👍